### ISSUE TITLE
Implement edgedb instance start for cloud instances

### ISSUE THREAD

**Issue by fmoor** [edgedb/edgedb-cli#1035]
`edgedb instance start` should work for cloud instances.

**Comment by tailhook** [comment]
Should this be no-op?

**Comment by fmoor** [comment]
I think cloud instances should be able to stop and start. @elprans do I have that right?

**Comment by elprans** [comment]
Yes, cloud instances should be able to go into a "stopped" state (i.e. not serving and not consuming any resources other than storage).

**Comment by fmoor** [comment]
I opened an issue for that functionality in nebula https://github.com/edgedb/nebula/issues/459
